pkg/pgmodel/extension.go

```diff
@@ -76,62 +76,70 @@ func checkPromscaleExtensionVersion(conn *pgx.Conn) error {
 	return nil
 }
 
-func migrateExtension(conn *pgx.Conn) error {
-	availableVersions, err := fetchAvailableExtensionVersions(conn)
-	ExtensionIsInstalled = false
+func migrateExtension(conn *pgx.Conn, extName string, extSchemaName string, validRange semver.Range, rangeString string) error {
+	availableVersions, err := fetchAvailableExtensionVersions(conn, extName)
 	if err != nil {
 		return err
 	}
 	if len(availableVersions) == 0 {
-		log.Warn("msg", "the promscale extension is not available, proceeding without extension")
-		return nil
+		return fmt.Errorf("the extension is not available")
```

Comment:
`the extension is not available` => `promscale extension unavailable`

Reply:
no because this now runs for both promscale and timescale

Reply:
Oh yes.